cypress/e2e/CMS/$HomeAndLandingPageTestCreationOfContent/index.js

```diff
@@ -0,0 +1,215 @@
+// <reference types="Cypress" />
+import { And } from 'cypress-cucumber-preprocessor/steps';
+function createRandomStr() {
+    var result = '';
+    var characters = 'abcdefghijklmnopqrstuvwxyz0123456789';
+    var charactersLength = characters.length;
+    for (var i = 0; i < 5; i++) {
+        result += characters.charAt(Math.floor(Math.random() * charactersLength));
+    }
+    return result;
+}
+let randomNum = createRandomStr();
+const siteSection = Cypress.env('test_site_section');
+const frontEndBaseUrl = Cypress.env('front_end_base_url');
+
+And('user fills out the following fields', (dataTable) => {
+    for (const { fieldLabel, value, field_name } of dataTable.hashes()) {
+        if (fieldLabel === 'Pretty URL') {
+            value = `${value}-${randomNum}`;
+        }
+        cy.get(`input[name^='${field_name}']`).as('inputField').parent().find('label').should('include.text', fieldLabel);
+        cy.get('@inputField').type(value);
+    }
+});
+
+And('user clicks on title with url {string} from the list of content', (contentHref) => {
+    cy.get(`a[href='${siteSection}/${contentHref}-${randomNum}']`).click();
+});
+
+And('user selects a checkbox next to title with url {string} from the list of content', (url) => {
+    cy.get(`a[href='${siteSection}/${url}-${randomNum}']`).parent().parent().find('input.form-checkbox').check();
+});
+
+Given('user is navigating to the front end site with path site section plus {string}', (purl) => {
+    cy.visit(`${frontEndBaseUrl}${siteSection}/${purl}-${randomNum}`, { retryOnStatusCodeFailure: true });
+});
+
+And('the content item with url {string} does not exist in the list of content', (url) => {
+    cy.get(`a[href='${siteSection}/${url}-${randomNum}']`).should('not.exist');
+});
+
+
+/*--------Scenario: User is adding new Home and Landing page content type and adding a Primary Feature Card Row -------*/
+And('user selects {string} from Contents dropdown',(dropDown) =>{
+    cy.get(`input[value="${dropDown}"]`).eq(0).click({force: true})
+})
+
+And('user clicks on {string} link in the {string} text area', (title, cartOption) =>{
+    cy.get(`div.paragraph-type-title:contains('${cartOption}')`).parent().parent().find(`span:contains('${title}')`).parent().click()      
+})  
+
+And('user clicks on {string} button item', (content) => {
+    cy.get(`input[value='${content}']`).click({force: true})
```

Comment:
now this step ( line 52)  is identical to step on line 88 
Again, make locators more specific - otherwise there is no guarantee it clicks on a right element ( at least from a code perspective)